src/main/resources/config/liquibase/changelog/20230908103500_expand_subject_demo_data_name_dob.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.6.xsd"
+>
+    <changeSet id="update person_name 20230908" author="bastiaan@thehyve.nl">
```

Comment:
you need to set that this only runs in the dev mode. To both changeSets, add attribute
```
 context="dev, test"
```